### PR TITLE
Create USDCEUR.yaml

### PR DESCRIPTION
**Pull request by ghost**
### Description

add USDCEUR client config

risk parameters are set to the same as `USDCUSD`

### Other changes

no

### Tested

tested by running the oracle client config locally

### Related issues

- Fixes part of https://github.com/mento-protocol/mento-general/issues/164

### Backwards compatibility

fully

### Documentation

- Analysis ipynb: https://colab.research.google.com/drive/1ccQOya2ExpZSB8SWRiED3_A61pwWu4gj#scrollTo=YBTMCkvOnZ1l
- data for choosing pairs: https://drive.google.com/drive/folders/1yqePlSOEbsV1GllDlQGQGzJeqgsItJc9

### CODE REVIEW

**Comment by ghost** [comment]
Deviations from average:
```
for 1 minute data since Dez 2022: deviations interval = [-0.005, 0.002]
for 30 minute data since Nov 2021: deviations interval = [-0.002, 0.006]
for 60 minute data since May 2021: deviations interval = [-0.004, 0.006]
```

I propose to set the max deviation risk parameter to the same as `USDCUSD`, 1%=0.01, that should cover the interval well including some buffer

**Review by sissnad — COMMENTED** [review]
Just wondering, the following route didn't give you a good price source?

```ts
[  
      {exchange: 'BINANCEUSD', symbol: 'USDCUSD', toInvert: false},
      {exchange: 'KRAKEN', symbol: 'EURUSD', toInvert: true}
 ]
 ```

**Comment by sissnad** [comment]
Otherwise it looks good to me!

**Comment by ghost** [comment]
> Just wondering, the following route didn't give you a good price source?
> 
> ```ts
> [  
>       {exchange: 'BINANCEUSD', symbol: 'USDCUSD', toInvert: false},
>       {exchange: 'KRAKEN', symbol: 'EURUSD', toInvert: true}
>  ]
> ```

increases deviations a bit but I'm good with adding as discussed. Pls see the latest commit

File: packages/helm-charts/oracle/USDCEUR.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+oracle:
+  currencyPair: USDCEUR
+  aggregation:
+    mid:
+      maxExchangeVolumeShare: 1
+      maxPercentageDeviation: 0.01
+      maxPercentageBidAskSpread: 0.005
+      # minReportPriceChangeThreshold
```
Comment (nvtaveras) [inline packages/helm-charts/oracle/USDCEUR.yaml]: can we remove this if not needed? also maybe from the `USDCUSD` file?

Comment (ghost) [reply packages/helm-charts/oracle/USDCEUR.yaml]: yes, it's not removed but moved to the end of the file in the reporter category

**Review by nvtaveras — APPROVED** [review]

**Pull request merged**